### PR TITLE
Feature/fogot password

### PR DESCRIPTION
**Pull request by FaysalSolutions**
What the title says.

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -1514,5 +1514,21 @@
...
+  "@forgotPassword": {
+    "description": "Forgot Password?"
+  },
+  "resetPassword": "Reset password",
+  "@resetPassword": {
+    "description": "Reset password"
+  },
+  "ifEmailCorrect": "If correct, an mail will be sent!",
+  "@ifEmailCorrect": {
+    "description": "Loading failed!"
```
Comment (mustiwebp) [inline lib/l10n/app_en.arb:1528]: Copy paste error

File: lib/screens/login_screen.dart [hunk 2]
```diff
@@ -180,6 +181,33 @@ class _LoginScreenState extends State<LoginScreen>
...
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: <Widget>[
+                    GestureDetector(
+                      onTap: () {
+                        Navigator.push(
+                          context,
+                          MaterialPageRoute<dynamic>(
+                            builder: (BuildContext context) {
+                              return const ForgotPasswordScreen();
```
Comment (mustiwebp) [inline lib/screens/login_screen.dart:195]: I would do this with our 'standard' route (cf. routes.dart), if possible.

File: lib/l10n/app_en.arb (hunk 1 above)
Comment (FaysalSolutions) [reply lib/l10n/app_en.arb]: Resolved [52663b8]

File: lib/screens/login_screen.dart (hunk 2 above)
Comment (FaysalSolutions) [reply lib/screens/login_screen.dart]: Resloved 9e6a0e1

**Review by mustiwebp — APPROVED** [review]

**Pull request merged**